Fedora-35/Dockerfile

```diff
@@ -56,15 +56,36 @@ RUN npm install -g npm markdownlint-cli cspell
 # This image is indented for jobs that run tests (and possibly also build)
 # firmware images. It is based on the build image and adds Qemu for the
 # architectures under test.
+
+#Building qemu from source:
 FROM build AS test
+ENV QEMU_URL="https://download.qemu.org/qemu-7.1.0.tar.xz"
 RUN microdnf \
       --assumeyes \
       --nodocs \
       --setopt=install_weak_deps=0 \
       install \
-        qemu-system-aarch64-core \
-        qemu-system-arm-core \
-        qemu-system-x86-core
+        bzip2 \
+        findutils \
+        git \
+        glib2-devel \
+        libfdt-devel \
+        ninja-build \
+        pixman-devel \
+        python3 \
```

Comment:
shouldn't python3 already be installed?

Reply:
The idea was to make the description of the sub-images somewhat
independent from each other.
Same situation with `tar`, which I add for qemu, but it is also added later in the dev image. I did not want to introduce more changes by removing it there.

I don't think adding packages twice does any harm (dnf figures that out anyway).